package.json

```diff
@@ -90,8 +90,8 @@
     "replace-in-file": "^4.0.0",
     "rimraf": "^3.0.2",
     "rollup": "^2.67.1",
+    "rollup-plugin-cleanup": "3.2.1",
     "rollup-plugin-license": "^2.6.1",
-    "rollup-plugin-re": "^1.0.7",
     "rollup-plugin-terser": "^7.0.2",
```

Comment:
we should change all of these to be hard pins - we can do it in another PR

Reply:
Can do!